### PR TITLE
Update jekyll-toc to v0.9.1 and add bd-callout to no_toc_section_class

### PR DESCRIPTION
**Pull request by toshimaru**
@XhmikosR Hi, I've notices you already upgraded jekyll-toc to v0.9.1 but it looks like you haven't changed `_config.yml` setting, so I did make a change to `_config.yml` as a sample.

Hopefully it works as you expected.

ref. https://github.com/toshimaru/jekyll-toc/issues/70

Thanks.

### CODE REVIEW

**Comment by XhmikosR** [comment]
Thanks, but the branch was just WIP. Plus I just replaced the headers yesterday with `h5` before you released 0.9.1. So I need to revert that here.

**Comment by XhmikosR** [comment]
@toshimaru: I think it looks good now, thanks for adding the feature :)

**Review by XhmikosR — APPROVED** [review]

File: site/_includes/callout.html [hunk 1]
```diff
@@ -4,6 +4,6 @@
   and type is one of: info (default), danger, warning
 {%- endcomment -%}
 {%- assign css_class = include.type | default: "info" -%}
-<div class="bd-callout bd-callout-{{ css_class }} no_toc_section">
+<div class="bd-callout bd-callout-{{ css_class }}">
```
Comment (toshimaru) [inline site/_includes/callout.html]: 👍

File: _config.yml [hunk 2]
```diff
@@ -62,4 +62,6 @@ cdn:
 toc:
   min_level:        2
   max_level:        4
-  no_toc_section_class: "bd-example"
+  no_toc_section_class:
+    - "bd-callout"
```
Comment (toshimaru) [inline _config.yml]: Nice!

**Pull request merged**